Detectors/DCS/testWorkflow/src/dcs-config-proxy.cxx

```diff
@@ -164,6 +177,10 @@ WorkflowSpec defineDataProcessing(ConfigContext const& config)
     dcsOutputs.emplace_back(DetID(id).getDataOrigin(), "DCS_CONFIG_FILE", 0, Lifetime::Timeframe);
```

Comment:
Seems we have a bug in all proxies: splitPayloadIndex must go from 0 to splitPayloadParts-1. And, for DCS to CCDB, where we send the multipart of 2, it should be 
```
    hdrN.splitPayloadParts = 2;
    hdrN.splitPayloadIndex = 0;
    hdrN.payloadSize = parts.At(0)->GetSize();
    hdrN.firstTForbit = 0; // this should be irrelevant for DCS


    hdrF.tfCounter = *timesliceId; // this also
    hdrF.payloadSerializationMethod = o2::header::gSerializationMethodNone;
    hdrF.splitPayloadParts = 2;
    hdrF.splitPayloadIndex = 1;
```
instead of 
https://github.com/AliceO2Group/AliceO2/blob/89e3fe068f17e371c1edc06ebcf688ad5cead9db/Detectors/DCS/testWorkflow/src/dcs-config-proxy.cxx#L100-L108

Reply:
Done